src/services/ViewService.js

```diff
@@ -171,22 +170,29 @@ function ViewService($q, $http, $rootScope, URLService, ElementService, UtilsSer
                     }
                 }
             }
-            if (view.specification) {
+            if (view.specification && view.specification.operand) {
                 var specContents = view.specification.operand;
                 for (var j = 0; j < specContents.length; j++) {
                     if (specContents[j] && specContents[j].instanceId) {
                         toGet.push(specContents[j].instanceId);
                     }
                 }
             }
-            toGetSet = new Set(toGet);
+            if (isTable(view) && view.specification && view.specification.value) {
+                try {
+                    var tableJson = JSON.parse(view.specification.value);
+                    if (tableJson.body) {
+                        collectTableSources(toGet, tableJson.body);
+                    }
+                } catch (e) {
+                }
```

Comment:
You should throw something, no?  Otherwise, you're making a silent error with this change that will be difficult to find when it fails (as it must, otherwise we wouldn't be "trying", right?).

Reply:
in this case we just want it to continue as a failed json parse isn't necessarily an error and is not critical

Reply:
well, I can't argue with that logic.  Though, I disagree with it whenever I see it, as it feels like we could stuff a json response of "failed to parse", without specifying actual error handling behavior and at no real cost. . .

I do like abdicating error handling, but I dislike wasting an opportunity to help the "me" that has to debug later.  ;)